libs/sdk-ui-dashboard/src/presentation/dashboardContexts/DashboardContentContext.tsx

```diff
@@ -0,0 +1,38 @@
+// (C) 2023 GoodData Corporation
+import React, { createContext, useContext } from "react";
+import { UnexpectedSdkError } from "@gooddata/sdk-ui";
+
+import { DashboardContentComponentProvider } from "./types.js";
+
+interface IDashboardContentContext {
```

Comment:
Can we use https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/dashboardContexts/DashboardComponentsContext.tsx#L48C7-L48C7 for this so we are not introducing additional context?

Reply:
Added to already existing context.